orca-clouddriver/orca-clouddriver.gradle

```diff
@@ -59,6 +59,7 @@ dependencies {
   testImplementation("io.strikt:strikt-core")
   testImplementation("io.mockk:mockk")
   testImplementation("ru.lanwen.wiremock:wiremock-junit5:1.3.1")
+  testImplementation 'org.mockito:mockito-inline:2.13.0'
```

Comment:
is the specific version needed? Would the dependency without version cause issue? Can you make the declaration syntax match with remaining i.e., adding parentheses and double quotes.

Reply:
Removed version number and formatted to match with the existing syntax